decidim-bulletin_board-ruby/.gitignore

```diff
@@ -15,4 +15,7 @@
 .rubocop-http*
 
 # byebug history
-.byebug_history
\ No newline at end of file
+.byebug_history
+
+# compiled dev JS client
+/app/assets/javascripts/decidim/bulletin_board/decidim-bulletin_board.dev.js
```

Comment:
Maybe we can ignore both artifacts since they are just included on the release. We need to ensure that the `gem build` task doesn't ignore them though.

Reply:
I've had some problems after deleting the .js, so that's why I've only added the .dev.js, but I agree with you. We can address that in future PRs.